File: mobile/src/main/java/com/alexstyl/specialdates/addevent/ToastDisplayer.java
package com.alexstyl.specialdates.addevent;

import android.content.Context;
import android.support.annotation.StringRes;
import android.widget.Toast;

final class ToastDisplayer implements MessageDisplayer {

    private final Context context;

    ToastDisplayer(Context context) {
        this.context = context;
    }

    @Override
    public void showMessage(@StringRes int stringResId) {
        Toast.makeText(context, stringResId, Toast.LENGTH_SHORT).show();
    }
}

File: android_mobile/src/main/java/com/alexstyl/specialdates/addevent/ToastDisplayer.kt
package com.alexstyl.specialdates.addevent

import android.content.Context
import android.widget.Toast

class ToastDisplayer(private val context: Context) : MessageDisplayer {

    override fun showMessage(string: String) {
        Toast.makeText(context, string, Toast.LENGTH_SHORT).show()
    }
}
